src/core/common/message.cpp

```diff
@@ -143,16 +147,18 @@ void MessagePool::FreeBuffers(Buffer *aBuffer)
     }
 }
 
-otError MessagePool::ReclaimBuffers(int aNumBuffers)
+otError MessagePool::ReclaimBuffers(int aNumBuffers, uint8_t aPriority)
 {
 #if OPENTHREAD_MTD || OPENTHREAD_FTD
     while (aNumBuffers > GetFreeBufferCount())
     {
```

Comment:
the condition should be `aNumBuffers  > otPlatMessagePoolNumFreeBuffers() ` when `OPENTHREAD_CONFIG_PLATFORM_MESSAGE_MANAGEMENT` is 1, right?

Reply:
Ignore this comment as I missed the fact that it is already capsulated in GetFreeBufferCount()